src/components/form-checkbox/form-checkbox.spec.js

```diff
@@ -147,47 +147,47 @@ describe('form-checkbox', async () => {
     expect(input.attributes('required')).not.toBeDefined()
   })
 
-  it('default has no name attribute on input', async () => {
+  it('has required attribute on input when prop required and name set', async () => {
     const wrapper = mount(Input, {
       propsData: {
-        checked: false
+        checked: false,
+        name: 'test',
+        required: true
       },
       slots: {
         default: 'foobar'
       }
     })
     const input = wrapper.find('input')
-    expect(input.attributes('name')).not.toBeDefined()
+    expect(input.attributes('required')).toBeDefined()
   })
 
-  it('default has name attribute on input when name prop set', async () => {
+  it('default has no name attribute on input', async () => {
     const wrapper = mount(Input, {
       propsData: {
-        checked: false,
-        name: 'test'
+        checked: false
       },
       slots: {
         default: 'foobar'
       }
     })
     const input = wrapper.find('input')
-    expect(input.attributes('name')).toBeDefined()
-    expect(input.attributes('name')).toEqual('test')
+    expect(input.attributes('name')).not.toBeDefined()
   })
 
-  it('default has required attribute on input when prop required set and prop name set', async () => {
```

Comment:
@tmorehouse Any idea why this test passed before?

Reply:
Hmmm

In the form-checkbox, (when not in a checkbox group) needs a name assigned to the single input in order to be required (HTML spec).

Reply:
Yeah, but before the changes of the PR the `required` prop of the input itself wan't applied.

Reply:
OR the test title might have been wrong...

Reply:
A single checkbox (not in a group) can have the required attribute, as long as a name is defined.

Reply:
Yeah for sure, but our current implementation didn't applied it and the test still passed.

Reply:
Weird